plymouthd.if

```diff
@@ -264,5 +264,13 @@ interface(`plymouthd_admin',`
 	admin_pattern($1, plymouthd_var_lib_t)
 
 	files_search_pids($1)
-	admin_pattern($1, plymouthd_var_run_t)
+		admin_pattern($1, plymouthd_runtime_t)
+
+	ifdef(`init_system',`
+		gen_require(`
+			class service { start status stop };
+		')
+
+		allow $1 plymouthd_unit_t:service { start status stop };
```

Comment:
Is it reasonable to allow the 4th argument to `init_startstop_service` to be empty?